docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        mixture = [
</pre><ul><li>better define <code>n_clusters = len(means)</code> beforehand and use it in the <code>range</code> or use list comprehension (<code>for scale, rng, number_sample, mean in zip(scales, rngs, number_samples, means)</code>)</li><li>Also, it seems that you always use the same number of samples everywhere so you don't need a list for it, just saying <code>number_samples = 8</code> below is fine.</li><li>similarly, you seem to only use one <code>scale</code>,no need for a list here.   </li><li>I think that <code>n_samples</code> is a better, most widely used way to call the variable.</li><li>Use consistent variable names between the function code and the call below.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I will code like that next time. For this notebook I used the GMM implementation of ott directly.